low-level/files/opt/seagate/sspl/setup/sspl_prepare.py

```diff
@@ -103,7 +103,7 @@ def validate(self):
         # Validate network interface availability
         for i_list in [mgmt_interfaces, data_private_interfaces, data_public_interfaces]:
             self.validate_nw_cable_connection(i_list)
-        self.validate_nw_interfaces()
+            self.validate_nw_interfaces(i_list)
```

Comment:
@cdeshmukh 
Instead of validating all the nw interfaces in the system, this change will only validate the interfaces provided in the input config files.

Reply:
yes, got it